patches/100_create_sms_fulfilment_tables.sql

```diff
@@ -0,0 +1,32 @@
+-- ***********************************************************************************************
```

Comment:
If we are going to run this as a patch, the patch version needs updating. But we might be okay to just ground zero this change again for the moment.

Reply:
Removed for now